### ISSUE TITLE
Commands "GetWaterLevel" and "GetWaterBoxInfo" have same output (T9)

### ISSUE THREAD

**Issue by unclej84** [mrbungle64/node-red-contrib-ecovacs-deebot#8]
Using commands "GetWaterLevel" and "GetWaterBoxInfo" results in two replies from the robot. Both return "WaterBoxInfo" and "WaterLevel". Is this similar for other models? Possibly one command could be removed if the output cannot be filtered to just get the requested information

**Comment by mrbungle64** [comment]
@unclej84 

> Is this similar for other models?

No, for XMPP and MQTT/XML based models the Ecovacs API provides 2 different commands:
* `GetWaterPermeability`
* `GetWaterBoxInfo`

As far as I know for MQTT/JSON based models you can only use `getWaterInfo`.

> Possibly one command could be removed if the output cannot be filtered to just get the requested information

I also would prefere to get the requested value only, but I don't think there is much added value either if I split the commands and also filter the ouput. Or do you see a lot of added value there?

**Comment by unclej84** [comment]
Interesting how different the models and communication protocols behave. As I assume there are currently very few people using this node - hopefully they become much more - I would not put any effort there for now. I can put a filter to the output by myself in the flow to just get the needed information.
